src/helper/Redux/sagas/reminder/reminder.saga.js

```diff
@@ -1,20 +1,52 @@
 import { takeLatest, call, put } from 'redux-saga/effects';
-import { setReminder } from '../../actions/reminder.action';
-import reminderService from '../../service/reminders.service';
-import { FETCH_REMINDER } from '../../types/reminder.types';
+import {
+  setReminder,
+  setReminderSuccess,
+  setReminderFailure,
+  fetchReminderSucess,
+  fetchReminderFailure,
+} from '../../actions/reminder.action';
+import {
+  FETCH_REMINDER,
+  FETCH_REMINDER_FAILURE,
+  FETCH_REMINDER_SUCCESS,
+} from '../../types/reminder.types';
+import { GetAllReminders } from '../../service/reminders.service';
+import { GetDataFromSnapshot } from '../../../Utils/GetDataFromSnapshot';
 
 // TODO: action calling the API
 function* fetchReminder() {
   try {
-    const reminders = yield call(reminderService.getAllReminders);
+    const reminders = yield call(GetAllReminders);
+    console.log(`reminders =>> ${reminders}`); // reminders =>> [object Object]
+
+    const resultDoc = GetDataFromSnapshot(reminders);
+
+    resultDoc
+      .then((data) => {
+        console.log(`resultDoc =>> ${resultDoc}`); // resultDoc =>> [object Promise]
+        console.log(`data =>> ${data}`);
+        // data =>> function () {
+        //   i.Zl(), r.cs.ws(function () {
+        //     return Pr(r.q_, o);
+        //   });
+        // }
+
+        // data.json();  // Unhandled Rejection (TypeError): data.json is not a function
+      })
+      .then((response) => console.log(`response =>> ${response}`)); // response =>> undefined
```

Comment:
The problem arises here when you see a log with each output that I put here for simplicity.
I expected the data as an object of JSON `data.json()` will easily be converted and get the result. But here will be something like callback fn this will be I didn't get the properly